app/controllers/api/v1/student_books_controller.rb

```diff
@@ -1,7 +1,8 @@
 class Api::V1::StudentBooksController < ApplicationController
   def create
+    # require 'pry'; binding.pry
     book = Book.find_or_create_by(book_params)
-    student_book = StudentBook.new(student_id: params[:student_id], book_id: book.id)
+    student_book = StudentBook.new(student_id: params[:student_id], book_id: book.id, prediction: params[:prediction])
```

Comment:
do we want to add reviews and review comments and status here as well then?

Reply:
We could add a status of 'reading' or 'not finished' if needed, but we should leave out reviews and review comments because students will need to complete the book to add those (they'll come in through a patch request)